installation/sql/mysql/base.sql

```diff
@@ -342,6 +342,7 @@ INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`,
 (0, 'plg_system_sef', 'plugin', 'sef', 'system', 0, 1, 1, 0, 1, '', '', '', 18, 0),
 (0, 'plg_system_sessiongc', 'plugin', 'sessiongc', 'system', 0, 1, 1, 0, 1, '', '', '', 19, 0),
 (0, 'plg_system_skipto', 'plugin', 'skipto', 'system', 0, 1, 1, 0, 1, '', '{}', '', 20, 0),
+(0, 'plg_system_shortcut', 'plugin', 'shortcut', 'system', 0, 0, 1, 0, 0, '', '{}', '', 0, 0),
```

Comment:
any reason not to enable by default on new installs?

Reply:
Not really. Thanks for the hint :)